libbeat/tests/system/requirements.txt

```diff
@@ -5,8 +5,8 @@ cached-property==1.4.2
 certifi==2018.1.18
 chardet==3.0.4
 deepdiff==4.2.0
-docker==4.1.0
-docker-compose==1.25.3
+docker==6.1.3
```

Comment:
the order does matter in the requirements.txt. Please move these on top

```
requests==2.31.0
urllib3==1.26.18
docker==6.1.3
docker-compose @ git+https://github.com/pkoutsovasilis/compose@v1_fix
```

Reply:
TIL, thanks